.github/workflows/deploy-enterprise-test.yml

```diff
@@ -23,19 +23,21 @@ jobs:
             deploy_ref: refs/heads/develop
             job_files: >-
               job_spec/AUTORIFT.yml
-              job_spec/INSAR_GAMMA.yml
```

Comment:
Might want to add `INSAR_GAMMA` back to `enterprise-test` for future convenience?